homeassistant/components/fritz/config_flow.py

```diff
@@ -107,6 +108,7 @@ def _async_create_entry(self) -> FlowResult:
             },
             options={
                 CONF_CONSIDER_HOME: DEFAULT_CONSIDER_HOME.total_seconds(),
+                CONF_OLD_DISCOVERY: False,
```

Comment:
Would define a `DEFAULT_CONF_OLD_DISCOVERY`, since this is used more than once ^^